frontend/packages/schema-editor/src/components/SchemaInspector/ItemFieldsTab.tsx

```diff
@@ -74,31 +74,32 @@ export const ItemFieldsTab = ({ selectedItem, language }: ItemFieldsTabProps) =>
   };
 
   const t = (key: string) => getTranslation(key, language);
+  const hasFieldNodes = !!fieldNodes.length;
 
   return (
     <div className={classes.root}>
-      {childNodes.length && (
+      {hasFieldNodes && (
```

Comment:
```suggestion
      {fieldNodes.length > 0 && (
```

Reply:
Thanks for reviewing my PR! I implemented your suggestion. :)